### PR TITLE
added clarification to simple compressible model

### PR DESCRIPTION
**Pull request by bobmyhill**
A few lines to explain that the simple compressible model doesn't obey Maxwell's relations.

### CODE REVIEW

File: include/aspect/material_model/simple_compressible.h [hunk 1]
```diff
@@ -31,9 +31,18 @@ namespace aspect
...
     /**
-     * A material model that consists of globally constant values for all
-     * material parameters except that the density decays linearly with the
-     * temperature and increases linearly with pressure.
+     * A material model that consists of globally constant values for the
+     * viscosity, thermal conductivity, thermal expansivity
+     * and isothermal compressibility. The density decays linearly with the
+     * temperature and increases exponentially with pressure.
+
+     * Note that this material model is not strictly self-consistent;
```
Comment (bangerth) [inline include/aspect/material_model/simple_compressible.h]: Add the asterisk also for the empty line.

File: include/aspect/material_model/simple_compressible.h [hunk 2]
```diff
@@ -31,9 +31,18 @@ namespace aspect
...
+     * the partial derivative of the density with respect to temperature
+     * at constant pressure is only equal to -alpha rho along the
```
Comment (bangerth) [inline include/aspect/material_model/simple_compressible.h]: can you use dollar-enclosed formulas here and below?

**Comment by bobmyhill** [comment]
Thanks @bangerth. I discussed this with @gassmoeller and @jdannberg, and I think we're on the same page. I added similar text to both the .cc and .h; is there any other file I need to add the text to?

File: source/material_model/simple_compressible.cc [hunk 3]
```diff
@@ -169,6 +169,12 @@ namespace aspect
                                    "  \\rho(p,T) = \\rho_0"
                                    "              \\left(1-\\alpha (T-T_a)\\right) "
                                    "              \\exp{\\beta (P-P_0))}"
-                                   "\\end{align}")
+                                   "\\end{align}"
+                                   "Note that this material model is not strictly self-consistent; "
+                                   "$\\partial \\rho / \\partial T |_P = -\\alpha \\rho$ only along the "
+                                   "reference isentrope, and "
+                                   "$\partial \rho / \partial P |_T = \beta \rho$ only if the isentrope "
```
Comment (bangerth) [inline source/material_model/simple_compressible.cc]: Almost. You need to double up the backslashes here.

**Comment by bangerth** [comment]
I'll let @gassmoeller make the call here.

**Comment by bobmyhill** [comment]
I've rephrased the comments to be more pragmatic, and will add a couple of paragraphs to the ASPECT manual to describe how to make thermodynamically self-consistent material models.

**Comment by bobmyhill** [comment]
@gassmoeller, @jdannberg, is this better now?

**Comment by bangerth** [comment]
Let's go with this. You were also willing to write a section for the manual about these issues in general, right? If so, can you open a github issue that reminds us to do this?

**Pull request merged**